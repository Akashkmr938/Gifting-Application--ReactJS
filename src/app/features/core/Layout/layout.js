import React from 'react';
import LayoutStyles from './layout.module.scss';
import Routes from '../../../routes/routes';

const Layout = () => {
    return (      
            <div className={LayoutStyles['parent-wrapper']}>
                <Routes />
            </div>
    )
}

export default Layout;