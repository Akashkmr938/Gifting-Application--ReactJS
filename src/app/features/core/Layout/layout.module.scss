.parent-wrapper {
    text-align: -webkit-center;
    margin-top: 20px;
    min-height: 71.5vh;
    overflow: hidden;
}