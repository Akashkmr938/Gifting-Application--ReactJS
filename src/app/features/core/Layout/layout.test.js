import React from "react";
import renderer from "react-test-renderer";
import Layout from "./layout";
import { MemoryRouter } from "react-router";

describe("The Layout component", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Layout />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
