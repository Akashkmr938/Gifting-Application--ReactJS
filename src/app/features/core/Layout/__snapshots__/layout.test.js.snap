// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Layout component renders correctly 1`] = `
<div
  className="parent-wrapper"
>
  <div
    className="MuiCircularProgress-root makeStyles-progress-1 MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
    role="progressbar"
    style={
      Object {
        "height": 40,
        "width": 40,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
        cx={44}
        cy={44}
        fill="none"
        r={20.2}
        strokeWidth={3.6}
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
